### ISSUE TITLE
Feat: Add support for plain text response as alternative to JSON

### ISSUE THREAD

**Issue by stelioschar** [micromata/http-fake-backend#7]
Hey @mischah,
Very promising work! I landed here once the json-server has some limitations.

What I would like to know is how can I return a simple text in some cases. 

E.g When I make a PUT request to `/api/simpleExample/update/1` to get the text `DONE`

```javascript
module.exports = SetupEndpoint({
    name: 'simpleExample',
    urls: [{
        requests: [...]
    },
    {
        params: '/update/{id}',
        requests: [{
            method: ['PUT', 'PATCH'],
            response: 'DONE'
        }, {
            method: 'DELETE',
            response: 'DELETED'
        }]
    }]
});
```
Thank you

**Comment by mischah** [comment]
Hej, this is something which:
- is already on my imaginary feature list
- I have a plan how to implement it
  - It will cause a breaking change

I wish I had more time, but I already spent my open source time slot for today otherwise I would start coding just now.

Long story short:
I won’t promise anything, but it will be the next feature which land very soon.

Cheers, Michael

P.S.
I wrote an article about the pros **and** cons compared to json-server which might be interesting: http://michael-kuehnel.de/api/2016/11/04/data-mocking-ways-to-fake-a-backend-api.html#pros-3
